http/src/main/resources/reference.conf

```diff
@@ -975,3 +975,8 @@ app-service {
 drs {
   url = "https://drshub.dsde-dev.broadinstitute.org/api/v4/drs/resolve"
 }
+
+metrics {
+  check-interval = 5 minutes
+  include-azure-cloud-context = true
```

Comment:
Normally it's not good to add metrics for things with high cardinality. We wouldn't want to include a tag for `workspaceId`, for example. However MRGs are pretty low cardinality, and per-MRG metrics would be extremely useful for public preview launch. So I'm including it as a tag, but making it configurable.

Reply:
Yeah sounds good! Will do.

Also crossed my mind to make the `LeoMetricsMonitor` as a whole disable-able -- just in case of disaster. :)